### PR TITLE
Issue/ccp 104

### PR DESCRIPTION
**Pull request by nischalsamji**
User's module fixes

### CODE REVIEW

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/validation/ValuesMatchValidator.java [hunk 1]
```diff
@@ -1,38 +1,85 @@
...
-		}
-		return true;
-	}
+public class ValuesMatchValidator implements ConstraintValidator<ValuesMatch, Object> {
+
+    private String firstFieldName;
+    private String secondFieldName;
+
+    @Autowired
+    Logger logger;
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/validation/ValuesMatchValidator.java]: private

File: Conceptpower+Spring/src/main/resources/validatormessages_en_US.properties [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+exists.email = A User with this email address already exists. Please choose a different email address.
+proper.username = Username cannot have any special characters or capital letters.
+proper.name = Name cannot have any special characters. Only spaces allowed.
+proper.email = Please give a proper email address.
+match.passwords = Passwords do not match.
+#Concept List Creation
+concept_name.required = Concept name is mandatory
+concept_description.required = Concept Description is mandatory
+concept_unique.required = Concept name should be unique
+concept_name.wordnet = Concept name should not be WordNet
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/resources/validatormessages_en_US.properties]: add periods to the end of each sentence.

**Comment by jdamerow** [comment]
just some tiny things

**Comment by nischalsamji** [comment]
@jdamerow done

**Pull request merged**